src/main/java/com/dyonovan/tcnodetracker/lib/JsonUtils.java

```diff
@@ -3,15 +3,62 @@
 import com.dyonovan.tcnodetracker.TCNodeTracker;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
 import com.google.gson.reflect.TypeToken;
 import java.io.*;
+import java.lang.reflect.Type;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.format.FormatStyle;
 import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class JsonUtils {
 
+    private static Logger LOGGER = LogManager.getLogger("TCNodeTracker");
```

Comment:
Probably, this logger should be moved to main mod class (with `@Mod`). To prevent creation of additional loggers, if ones would be needed. And use `Constants.MODNAME` or `Constants.MODID` for registering logger.